pages/students.tsx

```diff
@@ -76,35 +76,35 @@ export default function StudentsPage() {
         <div>
           <ul>
           <li>
-            <Link href="/" passHref>Deffered Placement Policy</Link>
+            <Link href="public/docs/2021-22/deferred_placement_policy.pdf" passHref>Deffered Placement Policy</Link>
```

Comment:
Don't need `public/` anywhere, just the sub-folder. Linking will not work this way.